Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/azure/token.go

```diff
@@ -1,12 +1,19 @@
 package azure
 
 import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha1"
+	"encoding/base64"
+	"fmt"
 	"net/http"
 	"net/url"
 	"strconv"
 	"time"
 
 	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/golang.org/x/crypto/pkcs12"
+	"github.com/dgrijalva/jwt-go"
```

Comment:
Actually, I take that back.

The test dependencies are included regardless of specifying `-t` to Godep.

The only reason it wasn't vendored before is because we were only using `arm/` and `management/` as sources of things to vendor. Now that we vendor everything in `./...`, it looks at additional files that include gocheck.

I think this is actually the right thing to do. It's a dependency of our repo. If they vendor our SDK, they should still be able to edit the code and run the tests; that's part of the benefit of vendoring. 

Further, the only way to undo adding gocheck is to go back to excluding parts of the project from Godep's control... which I really think is the wrong thing to do.